.gitignore

```diff
@@ -7,17 +7,31 @@
 /.foreman
 /coverage/*
 /db/*.sqlite3
+
 /log/*
 /tmp/*
 !/log/.keep
 !/tmp/.keep
+
 /spec/example_app/log/*
 /spec/example_app/tmp/*
 !/spec/example_app/log/.keep
 !/spec/example_app/tmp/.keep
 /spec/example_app/public/system
 /spec/example_app/public/assets
+
+/spec/example_app/app/assets/builds/*
+!/spec/example_app/app/assets/builds/.keep
+
+/spec/example_app/node_modules
 /spec/example_app/config/database.yml
+
+/node_modules
+/public
+
+/app/assets/builds/*
+!/app/assets/builds/.keep
```

Comment:
Doing more testing, I have found a reason to not gitignore this folder: if the compiled assets are not committed, you'll get the scss error if you pull the gem directly from the repo. Eg: `gem "administrate", github: "thoughtbot/administrate", branch: ...`.

Reply:
You're right, if we don't commit the compiled assets we lose the ability to point to a specific branch/commit directly in the Gemfile. I'll update to commit the compiled assets

Reply:
Updated in https://github.com/thoughtbot/administrate/pull/2397/commits/fb9beff9544445eef71f106881073e2a4a8214c6

That adds 10k lines from the compiled CSS and JS :smiley: